packages/google_mobile_maps/android/src/main/java/io/flutter/plugins/googlemobilemaps/GoogleMobileMapsPlugin.java

```diff
@@ -268,8 +371,46 @@ void hideOverlay() {
     parent.addView(mapView, 0);
   }
 
-  void moveCamera(CameraUpdate cameraUpdate) {
-    googleMap.animateCamera(cameraUpdate);
+  void moveCamera(final CameraUpdate cameraUpdate) {
+    if (googleMap == null) {
+      pendingOperations.add(
```

Comment:
We might want to save the pre-initialized operations on the dart side to avoid duplicating this logic for ios.

Reply:
Done.